src/asm/kane_pull_request

```diff
@@ -0,0 +1,25 @@
+03/01/2018 / 03/02/2018
+	-removed get_next_line.c file, asm.h reference/struct, Makefile reference
```

Comment:
Put this in the commit message, not in a file!

`git config --global core.editor vim` or your preferred text editor to write long commit messages.